subprojects/enterprise/enterprise.gradle.kts

```diff
@@ -27,6 +27,8 @@ dependencies {
     implementation(project(":core-api"))
     implementation(project(":core"))
     implementation(project(":launcher"))
+    implementation(project(":file-collections"))
+    implementation(project(":snapshots"))
```

Comment:
❓ Are these dependencies still needed?

Reply:
Good point! I've removed the dependency to `:file-collections`. The other is still required because that's where the `FileSystemAccess` service is located.

FIXED (5426e599)